ext/wmq_queue.c

```diff
@@ -772,28 +775,33 @@ VALUE Queue_get(VALUE self, VALUE hash)
               &pq->comp_code,      /* completion code                   */
               &pq->reason_code);   /* reason code                       */
 
-        /* report reason, if any     */
-        if (pq->reason_code != MQRC_NONE)
+        if (pq->reason_code == MQRC_TRUNCATED_MSG_FAILED)
```

Comment:
Looks like this the `pq->reason_code != MQRC_NONE` check. Is that intended?

Reply:
Yes. The previously logic was:
```
if (pq->reason_code != MQRC_NONE)
    if (pq->reason_code == MQRC_TRUNCATED_MSG_FAILED)
```